### PR TITLE
Fix multiple media

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

File: lib/parse-imports.js [hunk 1]
```diff
@@ -1,6 +1,22 @@
 var valueParser = require("postcss-value-parser")
 var stringify = valueParser.stringify
 
+function getMedia(params) {
+  var media = []
+  var last = params.reduce(function(medium, node, index) {
+    if (index < 2) {
+      return ""
+    }
+    if (node.type === "div" && node.value === ",") {
```
Comment (MoOx) [inline lib/parse-imports.js]: Why is the a test on a "div" ? Not sure to follow here.

Comment (TrySound) [inline lib/parse-imports.js]: div is divider (',', '/', ':')
This function just split params with comma divider after second (url+space) node

Comment (TrySound) [inline lib/parse-imports.js]: The reason is
`(cond1 + cond2) * parentCond = cond1 * parentCond + cond2 * parentCond`

Comment (MoOx) [inline lib/parse-imports.js]: Wow, "div" is confusing.

**Comment by MoOx** [comment]
Good job.

**Pull request merged**